backend/app/analysis/pop_density_detection.py

```diff
@@ -0,0 +1,55 @@
+"""
+
+pop_density_detection.py - analysis to count the number of people in the photos in the DB
+
+"""
+from __future__ import print_function
```

Comment:
Please fix order of imports (grouped by first Python imports, then 3rd party, then Django imports, then our project imports)

Reply:
We made changes for all of the comments.
We updated the order of imports, but we're not completely sure it is correct.